.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Thanks for your contribution. It must be reviewed by a member of the [`cockroachdb/docs-ext-pr-reviewers`](https://github.com/orgs/cockroachdb/teams/docs-ext-pr-reviewers) team. Before asking for review, add a description of your change and complete the following (add an `x` between the brackets):
+
+- [ ] Successful local build (`make standard`)
+- [ ] Successful local linkcheck (`make linkcheck`)
+- [ ] Successful pull request tests
+
+To view a HTML preview of each modified page, look for the **Files Changed** comment from the **github-actions** bot. Unfortunately, previews are not available if you use a fork.
```

Comment:
I thought external contributors can't work via a branch of the repo, only via a fork. Am I wrong? Is the idea that this text works for employees outside our team as well as external contributors?

Reply:
My understanding was that forks are not yet required for anyone. If that's incorrect, then I might need to make some tweaks.

The intention was to give helpful context for any contributors (inside or outside CRL) who may be less familiar with our processes and expectations for pull requests.

Reply:
Forks are required for external contributors. Only docs team uses the main branch at this point. I hope for all of us to move to forks eventually.

Reply:
(just a reminder that our changed files action does not work for forks)

Reply:
I can't believe I missed that. I should have known. Thanks for the reminder!